lib/logging/post_auth.rb

```diff
@@ -39,7 +40,7 @@ def update_user_last_login(username)
       user = User.find(username: username)
       return unless user
 
-      user.last_login = Time.now.strftime('%y-%m-%d %H:%M:%S')
+      user.last_login = Time.now.strftime('%Y-%m-%d %H:%M:%S')
       user.save
     end
```

Comment:
I thought `Sequel` gem will autoformat data according to its type... won't just `last_login = Time.now` work?

Also, `Time.now` is a system time but since we have servers in the same timezone, I guess that doesn't affect us.

Reply:
Wasn't aware that Sequel would do this, removing the strftime then.

Reply:
I'd check though 'cos my memory might be wrong 👍